naviswarm/ScanObs scanObs
naviswarm/GoalObs goalObs
naviswarm/ActionObs actionObs
naviswarm/VelocityObs velObs
naviswarm/CameraImageObs ImageObs